crates/std_detect/src/detect/arch/riscv.rs

```diff
@@ -81,13 +80,11 @@ features! {
     ///
     /// Defined by Privileged Specification:
     ///
-    /// * Supervisor: `"s"`
+    /// * *Supervisor-Level ISA* (not "S" extension): `"s"`
+    /// * H (hypervisor): `"h"`
     /// * Svnapot: `"svnapot"`
     /// * Svpbmt: `"svpbmt"`
     /// * Svinval: `"svinval"`
```

Comment:
I still think we should just remove privileged features entirely. They're unlikely to pass stabilization.

Reply:
In fact, #1770 actually removes this.

My intentions of this change are (1) like putting makeup on a dead (before its funeral) and (2) ensure that the documentation gets better even in case if PR #1770 (which actually removes this line) stalls for some reason.